data/rulei/rulei missions.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2022 by Saugia
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Relic of the Past"
+	invisible
+	landing
+	npc kill
+		government "vyu-Rulei"
+		personality heroic vindictive staying
+		system Kanguwa
+		ship "vyu-Ira" "Solemnity"
+
+#Future Conditions
```

Comment:
Yep. I'm leaving it in for now to see what Derpy has to say about whether or not I add it in this PR. It would be entirely a conversational mission, but it has merit being separate from this PR.